connector/connect/src/main/protobuf/spark/connect/base.proto

```diff
@@ -51,6 +52,12 @@ message Request {
   message UserContext {
     string user_id = 1;
     string user_name = 2;
+
+    // To extend the existing user context message that is used to identify incoming requests,
+    // Spark Connect leverages the Any protobuf type that can be used to inject arbitrary other
+    // messages into this message. Extensions are stored as a `repeated` type to be able to
+    // handle multiple active extensions.
+    repeated google.protobuf.Any extensions = 999;
```

Comment:
Just for my self education: 

Is Any type potentially a cause of security holes?

Reply:
No any is just used to define extension points in the proto so that at the point of writing the proto we don't need to define message that is embedded in the proto.

Interpreting the serialized class is up to the consumer and follows the same security model as any other serialized code so we have to be careful but it's not a new threat.

Reply:
I see. Makes sense.